docs/editor/codebasics.md

```diff
@@ -206,6 +206,40 @@ Example:
 
 The modifiers can also be stacked - for example, `\u\u\u$1` will uppercase the first three characters of the group, or `\l\U$1` will lowercase the first character, and uppercase the rest. The capture group is referenced by `$n` in the replacement string, where `n` is the order of the capture group.
 
+## Search Editor
+Search Editors let you view workspace search results in a full-sized editor, complete with syntax highlighting and optional lines of surrounding context.
+
+Below is a search for the word 'SearchEditor' with two lines of text before and after the match for context:
+
+
+![search editor overview](images/codebasics/search-editor-overview.png)
+
+The Open Search Editor command opens an existing search editor if one exists, or to otherwise create a new one. The pre-existing command Open new Search Editor has been renamed to New Search Editor, and will always create a new Search Editor.
+
+
+In the Search Editor, results can be navigated to using Go to Definition actions, such as F12 to open the source location in the current editor group, or Ctrl+K F12 to open the location in an editor to the side. Additionally, double-clicking can optionally open the source location, configurable with the `search.searchEditor.doubleClickBehaviour` setting.
+
+![search editor triggers](images/codebasics/search-editor-triggers.gif)
+
+### Search editor command arguments
+* `search.action.openNewEditor` - Opens the search editor in a new tab.
+- `search.action.openInEditor` - Copy the current Search results into a new Search editor
+* `search.action.openNewEditorToSide` - Opens the search editor in a new window next to the window you currently have opened.
+
+You can also use the Open New Search Editor button at the top of the Search view, and can copy your existing results from a Search view over to a Search Editor with the <b>Open in Editor link</b> added to the top of the results tree, or the <b>Search Editor: Open Results </b> in Editor command.
+
+![Search Editor Button](images/codebasics/search-editor-button.png)
+
+Search editor was opened by clicking the Open New Search Editor button (third button) on the top of the search view
+
+Note: Search Editors are enabled by default in [Insiders](https://code.visualstudio.com/insiders/), and can be opted into in Stable by setting search.enableSearchEditorPreview to true
```

Comment:
This setting no longer exists and was only available while Search Editors were in preview.
Please check all settings and command IDs to make sure they are still available in Stable.

Reply:
How do I check all settings and command IDs to make sure they are still available in Stable? Also do I leave this line?

Note: Search Editors are enabled by default in [Insiders]

Reply:
I think I checked everything: do I look in Open Default Keyboard Shortcuts(JSON)?

Reply:
You can remove the mention of Insiders since Search Editors are enabled by default on Stable (the regular VS Code builds).

You can check settings in the Settings editor (File > Preferences > Settings) and search by setting ID.
You can check commands in the Keyboard Shortcut editor (File > Preferences > Keyboard Shortcuts) and search by command ID.

Reply:
Great, thanks for checking all the settings and command IDs.

Reply:
Thank you for all the help and guidance on this PR. Really glad I can contribute! If there is anything else you think I can help out with please let me know :)